Readme.md

```diff
@@ -98,16 +98,16 @@ Note that all citations are in [BibLaTeX](https://ctan.org/pkg/biblatex) format.
 
 `Manopt.jl` belongs to the Manopt family:
 
-*  [manopt.org](https://www.manopt.org) – The Matlab version of Manopt, see also their :octocat: [GitHub repository](https://github.com/NicolasBoumal/manopt)
-* [pymanopt.org](https://www.pymanopt.org/) – The Python version of Manopt – providing also several AD backends, see also their :octocat: [GitHub repository](https://github.com/pymanopt/pymanopt)
+*  [manopt.org](https://www.manopt.org)—The MATLAB version of Manopt, see also their :octocat: [GitHub repository](https://github.com/NicolasBoumal/manopt)
```

Comment:
MATLAB is capitalized.

Reply:
Again, does not look nice, but yes, it is how they write their name so it might have to be like that. To me I read that always in a screaming and loud voice if it is caps.

Reply:
The registered mark is "MATLAB" so we don't have much choice over what looks nice.